src/snyk/common/analytics/itly.ts

```diff
@@ -5,9 +5,10 @@ import { v4 as uuidv4 } from 'uuid';
 import { ILog } from '../logger/interfaces';
 import itly, {
   AnalysisIsReadyProperties,
+  
   AnalysisIsTriggeredProperties as _AnalysisIsTriggeredProperties,
   IssueIsViewedProperties,
-} from '../../../itly';
+} from '../../../ampli';
```

Comment:
I was considering that but decided to postpone this work because we are still dependent on @itly packages in our dependencies in `package.json`. I've followed the same principle as `registry` to keep the naming for now. It seems Amplitude folks have to do the renaming work or correctly communicate difference between itly and ampli.

The change in this PR relates only to the CLI part, not the SDK.